README.md

```diff
@@ -67,6 +67,32 @@ variables. Example:
 
 ## Docker
 
+### Running pre-build docker image
+
+Official docker images are build and hosted on the
```

Comment:
s/build/built/

Also either drop "the" or change the link text to "Docker Hub".